src/frontend/templates/account/email/base_message.txt

```diff
@@ -0,0 +1,7 @@
+{% load i18n %}{% autoescape off %}{% blocktrans with site_name=current_site.name %}Hello from {{ site_name }}!{% endblocktrans %}
```

Comment:
Shouldn't we do html email ?

You can use https://mjml.io/try-it-live/templates/recast and edit on the left, on the right you will see the email and an html that is well supported on mail platforms.

Reply:
I'm not going to consider this an important change to make and test. All I did with this base message is to use django_allauth's template and slightly adjust the message - other than a slightly customized wording, this is exactly the default template from django_allauth.

Reply:
Fine, later on you (or me) might consider adding html version too https://django-allauth.readthedocs.io/en/latest/advanced.html#sending-email